gridstatus/pjm.py

```diff
@@ -554,25 +554,49 @@ def get_load_forecast(self, date, verbose=False):
             ),
             "forecast_area": "RTO_COMBINED",
```

Comment:
while we're in this method, i think we should update it to get all forecast areas. to be consistent with how we do load, I think we should pivot on `forecast_area` make them each unique columns

Reply:
im noticing tht the 7 day forecast has different regions than the historical

<img width="723" alt="image" src="https://github.com/kmax12/gridstatus/assets/371792/6eab7aa1-21e3-4105-98ce-2f0b82601dab">

vs 

<img width="585" alt="image" src="https://github.com/kmax12/gridstatus/assets/371792/6643d4c0-3c9a-40de-be73-ca3f4a0bd47a">

Reply:
That's a good callout. I only checked that the load forecast for `RTO` in the historical data from the most recent vintage was ≈ `RTO_COMBINED` in the seven day forecast.

@cwaldoch Do you know why PJM would have different forecast areas in the two datasets?

Reply:
> I think we should pivot on forecast_area make them each unique columns

IMO it's better to have a `forecast_area` column instead of making each area a column if you are storing the data because otherwise you have to update your table every time the forecast areas change. It depends on the use case though.

Reply:
You're right about it being more flexible format.

I don't think forecast areas change very often, which I why I did it this for load. I think for end users the columns is an easier to use format too. I mainly was trying to be consistent with how we do the load. 

I'm okay if we want a forecast_area column in this case.

Reply:
The same areas also named differently across the load forecast datasets